### PR TITLE
docs: Bring docs up to date for 1.19

### PR DESCRIPTION
**Pull request by matmen**
- [x] update screenshots (todo: [notification screenshot](https://docs.fluidd.xyz/assets/images/notifications.png))
- [x] add screenshots for LDAP auth (#721)
- added documentation for object cancelation
  - [x] enable_object_processing (https://github.com/fluidd-core/FluiddPI/pull/46)
  - [x] document gcode-viewer updates (#754)

### CODE REVIEW

**Comment by matmen** [comment]
@pedrolamas Anything else that comes to mind that needs to be updated or changed? Full screen webcam feed is marked as `Docs - Needed` but I don't think that's really necessary?

**Comment by pedrolamas** [comment]
> Full screen webcam feed is marked as `Docs - Needed` but I don't think that's really necessary?

Agreed, I don't think that is required.

Nothing else comes to my mind in terms of pending docs updates for the release...

**Review by pedrolamas — APPROVED** [review]

**Comment by matmen** [comment]
I'll merge this after #754 is merged, just in case there's any more changes that need to be made

**Pull request merged**
